modules/fedimint-ln-server/src/lib.rs

```diff
@@ -1319,6 +1327,10 @@ mod tests {
         let server = Lightning::new(
             server_cfg[0].clone(),
             0.into(),
+            BitcoinRpcConfig {
+                kind: "esplora".to_string(),
+                url: "http://dummy.xyz".parse().unwrap(),
+            },
```

Comment:
Could we maybe set `kind` to something that definitely doesn't work/panics? Otherwise we run the risk that we keep using these values somewhere (even though the server returns gibberish, at least until someone buys `dummy.xyz` fo nefarious purposes).